cli/lib/kontena/cli/stacks/yaml/reader.rb

```diff
@@ -201,6 +205,10 @@ def execute(service_name = nil, name: loader.stack_name.stack, parent_name: nil,
 
         validate unless skip_validation
 
+        if is_compose? && internals_interpolated_yaml['version'].split('.').first != '2'
+          notifications << { 'file' => "Docker compose yaml version #{internals_interpolated_yaml['version']} is not fully supported" }
```

Comment:
Warning about supported compose versions might be a good idea, but this is now confusing me because both `kontena.yaml` and `docker-compose.yaml` files have a version field. Do we really want to warn about unsupported docker compose YAML versions if you you `kontena stack install kontena.yaml` with something like `version: '3.1'`? Should this only warn if you're actually loading a compose file with `extends`?